examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -65,8 +65,9 @@
    },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.            draws=int(5e3), chains=4, step=pm.Metropolis(), return_inferencedata=True
</pre><p><code>5000</code>  is shorter than <code>int(5e3)</code>  and I find it more clear</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>